source/tutorials/configure-selinux/configure-selinux.rst

```diff
@@ -0,0 +1,139 @@
+
+Add the initial sepolicy rules
+------------------------------
+
+#. Create a sub-folder '*rfkill*' under the *device/intel/project-celadon/sepolicy* directory for hosting the sepolicy rules files.
+
+#. Introduce the above folder to the sepolicy compiler by adding the following line to the board configuration overlay file '*device/intel/mixins/groups/rfkill/true/BoardConfig.mk*'.
+
+    .. code-block:: none
+
+        BOARD_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/rfkill
+
+#. Inside the sepolicy rules folder, create an initial file named '*file_contexts*' with the following content, to assign a file label for the *rfkill* executable file.
+
+    .. code-block:: none
+
+        /vendor/bin/rfkill-init.sh	u:object_r:rfkill_exec:s0
+
+#. Create a SELinux type enforcement file '*rfkill.te*' fo define the policy type and access control for the *rfkill* module:
+
+    .. code-block:: none
+        :emphasize-lines: 2, 4, 7
+
+        # Define a domain that rfkill process will run in
+        type rfkill, domain;
+        # Define a file type for the rfkill executable file, and assign this file type to it
+        type rfkill_exec, exec_type, file_type, vendor_file_type;
+        # Grant the permission to init process, so that the init process can start rfkill service
+        # from init.rc and transition to rfkill domain
+        init_daemon_domain(rfkill)
+
+Add the additional sepolicy rules
+---------------------------------
+
+5. It is seldom to add only initial sepolicy rules to make a module work properly. You should reboot the device and search any AVC errors related to the *rfkill* module in the system ``dmesg`` or ``logcat`` output. AVC errors are associated with **avc:** or **permissive=** keywords as below example.
+
+    .. figure:: images/sepolicy-rules.png
+        :align: center
+
+#. In the above output, the key '**scontext**' indicates which module generates the AVC error and the cause. You should then write sepolicy rules to the '*.te*' file associated with that module, according to the relevant AVC error as shown in the above example.
+
+Some useful tools for SELinux
+=============================
+
+* **audit2allow**
+    ``audit2allow`` is a host SELinux tool, which can be used to generate sepolicy rules based on the log file. It is available at *external/SELinux/prebuilts/bin/* folder on the host system.
+
+    .. code-block:: bash
+
+        $ adb shell logcat > my_log.txt
+        $ audit2allow -p patch_to_sepolicy -i my_log.txt -o generated_rules.txt
+
+* **load_policy**
+    ``load_policy`` is a tool on the Android device, which loads the compiled sepolicy at runtime, so you can validate the modified sepolicy rules without making a new build. It is available at */system/bin/* folder on the target platform.
+
+    To generate the precompiled sepolicy binary to be loaded by the ``load_policy`` command, go to the top-most |C| source tree:
```

Comment:
Suggested rewrite: 
To generate the precompiled sepolicy binary that is loaded with the ``load_policy`` command, go to the top-most |C| source tree: